bicep/main.bicep

```diff
@@ -1044,7 +1053,8 @@ var aks_addons = {
   }
   azureKeyvaultSecretsProvider: {
     config: {
-      enableSecretRotation: 'false'
+      enableSecretRotation: 'true'
```

Comment:
Are there scenarios where we would want this option set to false?

Reply:
I think we discussed this last week, as there was no downside to enabling (cost, performance, functionality), we thought it would be ok to always default on.